packages/apps/src/SideBar/index.tsx

```diff
@@ -197,14 +197,14 @@ export default styled(SideBar)`
   }
 
   .apps--SideBar {
-    align-items: center;
+    /* align-items: center;
```

Comment:
does commenting it means the style won't be applied to side bar? or we have over written it some where?

Reply:
Commented css does not work any more, I should have removed the ghost code, but leave it being commented in a bit to refer on where we made a change for now. 

The commented code is already moved inside the associated component, that is, component styles would be better to be defined inside `styled-components` Components, rather than defined somewhere with css classname.